node_package/src/serverRenderReactComponent.ts

```diff
@@ -99,7 +98,7 @@ as a renderFunction and not a simple React Function Component.`);
     renderingError = e;
   }
 
-  const consoleReplayScript = buildConsoleReplay();
+  const consoleHistoryAfterSyncExecution = console.history;
```

Comment:
Too confusing to use this getter...timing and globals

Could this be updated only for the pro version node renderer to return a object for the console history?

https://github.com/shakacode/react_on_rails/pull/1649/files#diff-cbb0f0518343f42b78a0ec373eddd1187ffb9275651ddb9758a1160c52176eb6R28-R34

```js
    const reactRenderingResult = createReactOutput({
      componentObj,
      domNodeId,
      trace,
      props,
      railsContext,
    });
```

Reply:
It's updated. Implementation is simpler now.